lib/phlex/csv.rb

```diff
@@ -100,10 +112,8 @@ def escape(value)
 		first_char = value[0]
 		last_char = value[-1]
 
-		if prevent_csv_injection? && FORMULA_PREFIXES[first_char]
-			# Prefix a single quote to prevent Excel, Google Docs, etc. from interpreting the value as a formula.
-			# See https://owasp.org/www-community/attacks/CSV_Injection
-			%("'#{value.gsub('"', '""')}")
+		if InjectionPolicy::Ignore != in_case_of_injection && FORMULA_PREFIXES[first_char]
```

Comment:
Don't we need to do the injection policy after potentially trimming whitespace?

Reply:
Oh nevermind, I see that we've trimmed whitespace already above. I didn't have the diff expanded and I thought it was happening in the `elsif`